shard.yml

```
@@ -0,0 +1,90 @@
+name: drivers
+version: 2.2.0
+
+# compile target
+targets:
+  test-harness:
+    main: src/app.cr
+  report:
+    main: src/report.cr
+
+dependencies:
+  action-controller:
+    github: spider-gazelle/action-controller
+    version: ~> 4.4
+
+  email:
+    github: arcage/crystal-email
+
+  exec_from:
+    github: place-labs/exec_from
+
+  inactive-support:
+    github: spider-gazelle/inactive-support
+
+  jwt:
+    github: crystal-community/jwt
+
+  pinger:
+    github: spider-gazelle/pinger
+
+  ntlm:
+    github: spider-gazelle/ntlm
+
+  link-header:
+    github: spider-gazelle/link-header
+
+  place_calendar:
+    github: PlaceOS/calendar
+
+  placeos-compiler:
+    github: placeos/compiler
+    version: ~> 3.5
+
+  placeos-driver:
+    github: placeos/driver
+    version: ~> 5.0
+
+  # The PlaceOS API client
+  placeos:
+    github: placeos/crystal-client
+    version: ~> 2.2
+
+  # HTTP Client helper
+  responsible:
+    github: place-labs/responsible
+
+  rwlock:
+    github: spider-gazelle/readers-writer
+
+  # Driver deps:
+  telnet:
+    github: spider-gazelle/telnet.cr
+
+  pars:
+    github: place-labs/pars
+    branch: master
+
+  # For lat lon location indexing
+  s2_cells:
+    github: spider-gazelle/s2_cells
+
+  qr-code:
+    github: spider-gazelle/qr-code
+
+  # For QR .png file export
+  stumpy_png:
+    github: stumpycr/stumpy_png
+
+  # the STOMP protocol
+  stomp:
+    github: spider-gazelle/stomp
+
+  # MQTT protocol
+  mqtt:
+    github: spider-gazelle/crystal-mqtt
+
+  # XML to JSON convertor
+  oq:
+    github: blacksmoke16/oq
+    version: ~> 1.2
```

```suggestion
  exec_from:
    github: place-labs/exec_from

  pinger:
    github: spider-gazelle/pinger

  placeos-compiler:
    github: placeos/compiler
    version: ~> 3.5
```

Most of the libs come in via the `drivers` repo itself. Very few of these libs are required by the spec runner itself.
I guessed which libs should be removed so you'll need to double-check that.